deployments/helm/hephaestus/templates/controller/cronjob.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: image-build-gc
```

Comment:
Should probably have a `{{ include "hephaestus.manager.name" . }}` prefix and the cronjob needs standard labels and annotations